custom_targets/upload_method_cfg/L452RE_CUSTOM_CLOCK.cmake

```diff
@@ -0,0 +1,12 @@
+## Example of custom uploud method for L452RE_CUSTOM_CLOCK
+
+
+# General config parameters
+# -------------------------------------------------------------
+set(UPLOAD_METHOD_DEFAULT OPENOCD)
+
+# Config options for OPENOCD
+# -------------------------------------------------------------
+set(OPENOCD_UPLOAD_ENABLED TRUE)
+set(OPENOCD_CHIP_CONFIG_COMMANDS
+    -f ${CMAKE_CURRENT_LIST_DIR}/openocd_cfgs/stm32l452re.cfg)
```

Comment:
Wait, I don't think this will work, because stm32l452re.cfg isn't actually in this repo

Reply:
Yes, that is good catch!  I will try to fix this somehow.